packages/data-context/src/sources/VersionsDataSource.ts

```diff
@@ -93,13 +92,13 @@ export class VersionsDataSource {
       this._currentTestingType = this.ctx.coreData.currentTestingType
       this.ctx.update((d) => {
         if (d.versionData) {
-          d.versionData.latestVersion = this.getLatestVersion()
+          d.versionData.latestVersion = this.#getLatestVersion()
         }
       })
     }
   }
 
-  private async getVersionMetadata (): Promise<Record<string, string>> {
+  async #getVersionMetadata (): Promise<Record<string, string>> {
```

Comment:
Trying to get some consistency by using `#` over `private`. I really think this I better for various reasons:

- `#` is a native JS; better than a proprietary feature
- `#` is actually private; `private` is just compile time private (I think)
- With `#`, you can see a method is private where it's called. With `private`, you need to look at the definition.

This file has both types, so I just the one I think is better and made them all the same.

Reply:
Today I learned I completely missed the fact that JS is gaining true private fields/methods. Cool!

Personally I've liked using Typescript's `private` in the past because I'm still able to use those items directly in unit tests which sometimes makes testing a bit cleaner/easier. But that's the lazy developer within me - if a class is hard to test using its public interface it's probably a hint that I haven't structured it very well

Reply:
TIL this also exists, very cool